src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/RebuildIndex_2.sql

```diff
@@ -0,0 +1,60 @@
+﻿/*************************************************************
+    Stored procedures for rebuild index
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     RebuildIndex
+--
+-- DESCRIPTION
+--     Stored procedures for rebuild index
+--
+-- PARAMETERS
+--     @tableName
+--         * index table name
+--     @indexName
+--         * index name
+--     @pageCompression
+--         * index page compression
+
+GO
+CREATE PROCEDURE [dbo].[RebuildIndex_2]
+    @tableName nvarchar(128),
+    @indexName nvarchar(128),
+    @pageCompression bit
```

Comment:
Please apply the same comment from the DisableIndex

Reply:
thanks, updated.